src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.cs

```diff
@@ -66,10 +70,30 @@ protected override void OnApplyTemplate()
 				}
 			}
 
+			//if (this.GetTemplateChild<Border>(c_expanderContentClip) is Border expanderContentClip)
```

Comment:
Was this disabled due to know issues? In my PR it is still enabled -- however, I wasn't able to test due to build issues.